test/services/database-analyzer/database-analyzer-sequelize.js

```diff
@@ -35,6 +36,31 @@ describe('Database analyser > Sequelize', () => {
         await sequelizeHelper.close();
       });
 
+      it('should give us constraint_name, table_name_ column_type, column_name, foreign_table_name, foreign_column_name, unique_indexes', async () => {
+        const tableForeignKeysAnalyzer = new TableForeignKeysAnalyzer(databaseConnection, 'public');
+        const constraints = await tableForeignKeysAnalyzer.perform('addresses');
+
+        if (dialect === 'postgres') {
+          expect(Object.keys(constraints[0]).sort()).to.eql([
```

Comment:
Only constraints[0] is covered by tests. What about the rest of the constraints array?